sydent/replication/peer.py

```diff
@@ -78,30 +78,33 @@ def __init__(self, sydent, server_name, pubkeys):
         super(RemotePeer, self).__init__(server_name, pubkeys)
         self.sydent = sydent
         self.port = 1001
+        self.alg = "ed25519"
```

Comment:
for ref: I think this would have made more sense as a global constant: `SIGNING_KEY_ALGORITHM = "ed25519"`, or something.

not important though

Reply:
I like that as well. Will update.